docs/Developer's Guide.md

```diff
@@ -0,0 +1,322 @@
+
+You are now able to point your local frontend to the remote backend by setting `REACT_APP_API_URL` to `localhost:[API_PC_PORT_NUMBER]`.
+
+# Code Style
+
+This project uses [Prettier](https://prettier.io/), an "opinionated code formatter," to automatically format JavaScript, JSON, Sass, and Markdown files according to a common style.
+
+The advantage of using a code formatter is consistency between developers. Using ESLint with a strict style guide is one approach to consistency, but it still requires developer effort to fix code according to the ESLint rules. Prettier takes care of this without any effort from the developers.
+
+Prettier is used as a pre commit hook in this repository. This means that it will automatically format any staged code before it is committed. It can also be used as an [extension for your editor](https://prettier.io/docs/en/editors.html). This repository includes a setup for VS Code: when the extension is installed, files will be formatted automatically every time they are saved.
+
+ESLint and Stylelint are used in conjunction with Prettier to catch syntax errors, but not to check code style - that is taken care of solely by Prettier.
+
+# File Organization
+
+The source files for the app are in `./src`. The other top-level folders are as follows:
+
+- `.github` contains configuration for GitHub Actions, which is used for maintenance, security, and CI/CD tasks.
+- `.husky` contains pre-commit hooks to enforce consistent formatting.
+- `.vscode` contains configuration for Visual Studio Code
+- `docs` contains supplemental documentation.
+- `build` contains the built application; not tracked by Git
+- `node_modules` contains dependencies installed by `npm`; not tracked by Git
+- `public` contains assets that should not be processed by Webpack. Only files inside `public` can be used from `public/index.html`.
+
+The structure of the `src` directory is as follows:
+
+```plain
+├── components
+│   └── ...
+├── contexts
+│   └── ...
+├── hooks
+│   └── ...
+├── services
+│   └── ...
+├── views
+│   └── ...
+├── app.js
+├── index.js
+└── pwa.js
+```
+
+## Components
+
+```plain
+components
+├── ...
+├── EventList
+│   ├── components
+│   │   └── ...
+│   ├── EventList.module.scss
+│   └── index.js
+└── ...
+```
+
+This folder contains components that, when used together, make up the views of the application. Each component should be small and focus on doing One Thing Well. Read about the [Atomic Design Methodology](http://atomicdesign.bradfrost.com/chapter-2/) for a useful perspective on writing small, reusable components and composing them into larger areas of functionality.
+
+Each component must have a folder named in PascalCase (also known as upper camel case) containing a file called `index.js`. Using that filename allows the component to be imported by folder, instead of by file: `import MyComponent from 'components/MyComponent` instead of `import MyComponent from 'components/MyComponent/my-component`.
```

Comment:
```suggestion
Each component must have a folder named in PascalCase format (also known as upper camel case) containing a file called `index.js`. Using that filename allows the component to be imported by folder, instead of by file: `import MyComponent from 'components/MyComponent` instead of `import MyComponent from 'components/MyComponent/my-component`.
```

Reply:
Okay, sure. The real issue in my mind is that "...folder named in PascalCase..." seems awkward to me. If you don't think so, fine to leave it, but perhaps 
```suggestion
Each component must have a folder with its name in PascalCase (also known as upper camel case) containing a file called `index.js`. Using that filename allows the component to be imported by folder, instead of by file: `import MyComponent from 'components/MyComponent` instead of `import MyComponent from 'components/MyComponent/my-component`.
```